html/lesson7/tutorial.md

```diff
@@ -0,0 +1,420 @@
+
+Mobile first design takes the opposite approach. The design process is started with consideration for mobile constraints and then you work your way up from there.
+
+<img src="assets/images/respond.png" style="max-width:500px">
+
+The mobile first approach aims to provide the best possible experience for mobile users by focussing on core content and functionality and by giving consideration to user's available bandwidth and likely needs.
+
+
+###Structure your content first
+
+The content of your site is the important stuff! Think about what you want your users to see and do when they get to your site. Create a hierachy of content that will get your information across to your users in a way that is meaningful and cohesive.
+
+In the case of our example, in order of importance, we've got the site name, the text, a list of useful offsite resources and info about the author. If we were to show these things on a mobile they'd appear in this order eg.
+
+![](assets/images/mobilelayout.png)
+
+So, lets get some markup written!  We're going to use our html5 elements to lay out the site so, put in a header, a section, a sidebar div and a footer with relevant class names
+
+    <header class="header">
+    </header>
+
+    <section class="about">
+    </section>
+
+    <div class="sidebar">
+    </div>
+
+    <footer class="footer">
+    </footer>
+
+This is the structure of our page.
+
+###Add in the content
+
+Now we're ready to flesh out our structure with content.  Looking at the example page, starting from the top we can see that the header contains an image and a heading, so lets put them into our HTML, with class names
+
+    <img class="header_img" src="assets/images/gracehopper_portrait.jpg" />
+    <h1 class="header_title"> Grace Hopper</h1>
+
+Next we have the copy in paragraphs interspersed with figures and a blockquote. Add this to your about section using
+
+    <p class="main_text">
+        ...
+    </p>
+
+    <figure class="figure">
+        <img class="figure_img" src="assets/images/grace.jpg">
+        <figcaption class="figure_caption">
+            ...
+        </figcaption>
+    </figure>
+
+    <blockquote class="main_quote">
+        ...
+    </blockquote>
+
+Next up is our list of resources and its title, this is going to be inside the sidebar and the HTML should look something like this (fill in the hrefs as you like)
+
+
+    <h2 class="resourcelist_title">Resources</h2>
+    <ul class="resourcelist">
+        <li class="resourcelist_item">
+            <a class="resourcelist_link" href="#">Computer Scientist</a>
+        </li>
+        ...
+    </ul>
+
+
+And finally we have the footer which will contain the author information. This will just be a p tag with an a. Give them appropriate class names eg
+
+    <p class="footer_text">Made by <a href="http://codebar.io" class="footer_link">codebar.io</a></p>
```

Comment:
in the above a element you have put the class="blah" after the href="blah" and here the other way round, could they be consistent please. This is small but could easily confuse a student. Thanks.